examples/compose/vtcompose/vtcompose.go

```diff
@@ -534,7 +534,7 @@ func generateDefaultShard(tabAlias int, shard string, keyspaceData keyspaceInfo,
   path: /services/init_shard_primary%[2]d
   value:
     image: vitess/lite:${VITESS_TAG:-latest}
-    command: ["sh", "-c", "/vt/bin/vtctlclient %[5]s InitShardPrimary -force %[4]s/%[3]s %[6]s-%[2]d "]
+    command: ["sh", "-c", "/vt/bin/vtctldclient %[5]s InitShardPrimary --force %[4]s/%[3]s %[6]s-%[2]d "]
```

Comment:
simply `-f` also works if we want to demo that in the examples

Reply:
```
vtctldclient InitShardPrimary -f
E1010 20:01:59.194594   92668 main.go:56] unknown shorthand flag: 'f' in -f
```

Reply:
ah shoot, sorry. we _should_ make that work (later)